lib/main.coffee

```diff
@@ -2,26 +2,35 @@
 {find} = helpers = require 'atom-linter'
 path = require 'path'
 globule = require 'globule'
+ChildProcess = require 'child_process'
```

Comment:
It would be more readable if we make this `{spawnSync}` as it would tell us at first look what is used in this file